lib/internal/fs/promises.js

```diff
@@ -739,6 +747,68 @@ async function readFile(path, options) {
   return PromisePrototypeFinally(readFileHandle(fd, options), fd.close);
 }
 
+async function stream(handle, dest, options) {
```

Comment:
If the function does the equivalent of `.pipe()`, then naming it `stream` is going to be confusing.